showcaseApp/build.gradle.kts

```diff
@@ -14,20 +16,22 @@ kotlin {
     sourceSets {
         commonMain.dependencies {
             implementation(compose.runtime)
-            implementation(compose.foundation)
+            api(compose.foundation)
```

Comment:
why is this API if this is the application?

Reply:
I'll check it